config/initializers/devise.rb

```diff
@@ -268,4 +268,10 @@
   config.otp_length = 6
   config.direct_otp_length = 6
   config.direct_otp_valid_for = 5.minutes
+
+  # zxcvbnable
+  # The scores 0, 1, 2, 3 or 4 are given when the estimated crack time (seconds)
```

Comment:
a link may be more helpful as a comment here since this comment block brings up more questions than it answers, for me :)